client/src/js/components/bhReportPeriodSelect.js

```diff
@@ -5,6 +5,7 @@ angular.module('bhima.components')
     // transclude  : true,
     bindings    : {
       periodId          : '<',
+      displayCurrentPeriod : '<?',
```

Comment:
This doesn't need to be an option passed in.  Just check if `periodId` is defined, and if not, default to the current period.